### PR TITLE
[TimeTests] Disable VPU option, rename cache option

### PR DESCRIPTION
**Pull request by just-sparta**
+ https://github.com/intel-innersource/frameworks.ai.openvino.ci.product-configs/pull/662

### CODE REVIEW

**Comment by just-sparta** [comment]
https://github.com/intel-innersource/frameworks.ai.openvino.ci.product-configs/pull/662

File: tests/time_tests/src/timetests/timetest_infer.cpp [hunk 1]
```diff
@@ -60,21 +48,19 @@ int runPipeline(const std::string &model, const std::string &device, const bool
...
             {
               SCOPED_TIMER(load_network);
-              exeNetwork = ie.LoadNetwork(cnnNetwork, device, vpuConfig);
+              exeNetwork = ie.LoadNetwork(cnnNetwork, device);
             }
           }
         }
         else {
-          SCOPED_TIMER(load_network);
+          SCOPED_TIMER(load_network_cache);
```
Comment (ArtemySkrebkov-intel) [inline tests/time_tests/src/timetests/timetest_infer.cpp:58]: Two questions:
* Why do we need to distinguish LoadNetwork for cache/non-cache? I am just wondering
* I do not really understand how models are put into the cache? The cache should be empty at first run of the tool, according to https://docs.openvino.ai/latest/openvino_docs_IE_DG_Model_caching_overview.html

Comment (just-sparta) [reply tests/time_tests/src/timetests/timetest_infer.cpp:58]: The same name of the metric for the cache and without can confuse

Working with the model cache directory is regulated in the runner scripts (we use pytest). There is no work with the cache directory inside tool at the moment

**Comment by just-sparta** [comment]
@nosovmik @vurusovs @ArtemySkrebkov-intel please review changes with time infer tool

File: tests/time_tests/src/timetests/timetest_infer.cpp [hunk 2]
```diff
@@ -16,40 +16,28 @@ using namespace InferenceEngine;
...
-
     // first_inference_latency = time_to_inference + first_inference
     {
       SCOPED_TIMER(time_to_inference);
       {
         SCOPED_TIMER(load_plugin);
-        if (performanceHint) {
-          TimeTest::setPerformanceConfig(ie, device);
-        }
+        TimeTest::setPerformanceConfig(ie, device);
```
Comment (vurusovs) [inline tests/time_tests/src/timetests/timetest_infer.cpp:32]: all configurations should be out of timer "load_plugin" (after the timer)

File: tests/time_tests/src/timetests_helper/cli.h [hunk 3]
```diff
@@ -26,17 +26,9 @@ static const char target_device_message[] =
...
-
 /// @brief message for cache argument
-static const char cpu_cache_message[] =
-    "Not required. Use this key to run timetests with CPU models caching.";
-
-/// @brief message for vpu argument
-static const char vpu_compiler_message[] =
-    "Not required. Use this key to run timetests with 'MLIR' or 'MCM' VPUX compiler type.";
+static const char model_cache_message[] =
+    "Not required. Use this key to run timetests with models caching.";
```
Comment (vurusovs) [inline tests/time_tests/src/timetests_helper/cli.h:31]: please, extend description with info that executable should be run twice - the second run will use cache prepared from first one

File: tests/time_tests/test_runner/conftest.py [hunk 4]
```diff
@@ -273,8 +251,8 @@ def prepare_db_info(request, test_info, executable, niter, manifest_metadata):
             test_info["db_info"].update(json.load(db_meta_f))
 
     # add cpu cache status
-    cpu_cache = True if request.config.getoption("cpu_cache") else False
-    test_info["db_info"].update({"use_cpu_cache": cpu_cache})
+    cpu_cache = True if request.config.getoption("model_cache") else False
```
Comment (vurusovs) [inline tests/time_tests/test_runner/conftest.py:254]: cpu_cache -> model_cache

Comment (vurusovs) [reply tests/time_tests/test_runner/conftest.py:254]: No need in if/else, `request.config.getoption("model_cache")` should return True/False

**Review by vurusovs — CHANGES_REQUESTED** [review]

File: tests/time_tests/src/timetests_helper/cli.h [hunk 5]
```diff
@@ -82,11 +66,7 @@ static void showUsage() {
...
-  std::cout << "    -d \"<device>\"             " << target_device_message
-            << std::endl;
-  std::cout << "    -s \"<path>\"               " << statistics_path_message
-            << std::endl;
-  std::cout << "    -p                          " << performance_hint_message << std::endl;
-  std::cout << "    -c                          " << cpu_cache_message << std::endl;
-  std::cout << "    -v \"<compiler_type>\"      " << vpu_compiler_message << std::endl;
+  std::cout << "    -d \"<device>\"             " << target_device_message << std::endl;
+  std::cout << "    -s \"<path>\"               " << statistics_path_message << std::endl;
+  std::cout << "    -c                          " << model_cache_message << std::endl;
```
Comment (nosovmik) [inline tests/time_tests/src/timetests_helper/cli.h:71]: final alignment on screen is a bit incorrect (as \" is actually one character " on a screen

**Review by nosovmik — APPROVED** [review]
Looks good from timetest_infer executable perspective. Tried different plugins, worked well

File: tests/time_tests/src/timetests/timetest_infer.cpp (hunk 2 above)
Comment (just-sparta) [reply tests/time_tests/src/timetests/timetest_infer.cpp]: done

File: tests/time_tests/src/timetests_helper/cli.h (hunk 3 above)
Comment (just-sparta) [reply tests/time_tests/src/timetests_helper/cli.h]: done

File: tests/time_tests/test_runner/conftest.py (hunk 4 above)
Comment (just-sparta) [reply tests/time_tests/test_runner/conftest.py]: done

File: tests/time_tests/src/timetests_helper/cli.h (hunk 5 above)
Comment (just-sparta) [reply tests/time_tests/src/timetests_helper/cli.h]: fixed

File: tests/time_tests/src/timetests_helper/cli.h [hunk 6]
```diff
@@ -77,16 +63,12 @@ DEFINE_string(s, "", statistics_path_message);
...
-  std::cout << "    -d \"<device>\"             " << target_device_message
-            << std::endl;
-  std::cout << "    -s \"<path>\"               " << statistics_path_message
-            << std::endl;
-  std::cout << "    -p                          " << performance_hint_message << std::endl;
-  std::cout << "    -c                          " << cpu_cache_message << std::endl;
-  std::cout << "    -v \"<compiler_type>\"      " << vpu_compiler_message << std::endl;
+  std::cout << "    -d \"<device>\"             " << target_device_message << std::endl;
+  std::cout << "    -s \"<path>\"               " << statistics_path_message << std::endl;
+  std::cout << "    -c                        " << model_cache_message << std::endl;
```
Comment (vurusovs) [inline tests/time_tests/src/timetests_helper/cli.h:73]: need to align spaces in help (aka tabulation)

**Review by vurusovs — APPROVED** [review]

**Review by ArtemySkrebkov-intel — APPROVED** [review]

**Pull request merged**